### PR TITLE
Create open_redirect_artisteer.yml

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

**Comment by aidenmitchell** [comment]
https://platform.sublime.security/hunts/1ff9dfa9-dc41-4cb8-b295-1d6b3cf24d6b

File: detection-rules/open_redirect_artisteer.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: "Open redirect: Artisteer"
+description: "Message contains use of the Artisteer open redirect, but the sender is not Artisteer. This has been exploited in the wild."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any(body.links,
+          .href_url.domain.root_domain == 'artisteer.com'
+          and strings.icontains(.href_url.query_params, 'redirect_url')
```
Comment (zoomequipd) [inline detection-rules/open_redirect_artisteer.yml:9]: looks like `p=affr` is required for the redirect to work.  
might also add `=` to the end of `redirect_url`

```
and strings.icontains(.href_url.query_params, 'redirect_url=')
and strings.icontains(.href_url.query_params, 'p=affr')
```

Examples: 
- doesn't work `http://www.artisteer.com/?redirect_url=https://example.com`
- doesn't work `http://www.artisteer.com/?p=AAAA&redirect_url=https://example.com`
- works `http://www.artisteer.com/?p=affr&redirect_url=https://example.com`

**Review by zoomequipd — CHANGES_REQUESTED** [review]
redirect is still active/working.

**Review by zoomequipd — CHANGES_REQUESTED** [review]
redirect is still active/working.

**Review by zoomequipd — APPROVED** [review]

**Pull request merged**